src/hex/net/common.rs

```diff
@@ -31,3 +33,16 @@ pub fn flip_score_if_needed(
         return (val, moves_probs);
     }
 }
+
+pub fn position_to_planes(pos: &HexPosition) -> Vec<Bitboard> {
+    let mut planes = Vec::new();
+    /* red pieces plane */
+    planes.push(pos.pieces_red());
+    /* blue pieces plane */
+    planes.push(pos.pieces_blue());
+    /* a plane with all ones to help NN find board edges */

```

Comment:
I thought there was supposed to be a plane with ones only in the edges?

Reply:
There is a plane with all ones. Added in another commit in the PR